Plugin/ConfigSectionPaymentPlugin.php

```diff
@@ -69,8 +69,12 @@ private function getKeys($configData)
     {
         $configFields = $configData['groups']['omise']['fields'];
 
-        $publicKeyIndex = ($configFields['sandbox_status']) ? 'test_public_key' : 'live_public_key';
-        $secretKeyIndex = ($configFields['sandbox_status']) ? 'test_secret_key' : 'live_secret_key';
+        $standboxStatus = array_key_exists('value', $configFields['sandbox_status'])
```

Comment:
```suggestion
        $sandboxStatus = array_key_exists('value', $configFields['sandbox_status'])
```
I guess you meant `$sandboxStatus`.

Reply:
@hpariwat Yes, thanks.